tests/test_other.py

```diff
@@ -2100,9 +2097,15 @@ def test_prepre(self):
   def test_save_bc(self):
     for save in [0, 1]:
       self.clear()
-      run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world_loop_malloc.cpp')] + ([] if not save else ['--save-bc', 'my_bitcode.bc']))
+      cmd = [PYTHON, EMCC, path_from_root('tests', 'hello_world_loop_malloc.cpp')]
+      if save:
+        cmd += ['--save-bc', 'my_bitcode.bc']
+      run_process(cmd)
       assert 'hello, world!' in run_js('a.out.js')
-      assert os.path.exists('my_bitcode.bc') == save
+      if save:
```

Comment:
This is more verbose but ultimately simpler to read

Reply:
Agreed.  It also means that back trace and line numbers within it will more accurately report what failed.